src/index.html

```diff
@@ -4,9 +4,318 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&amp;display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__ltem">
+            <a href="#Apple" class="is-active">Apple</a>
```

Comment:
It also `nav__link`

Reply:
<img width="680" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/70220667/336519b0-dc16-441a-a16d-5168f7551856">